.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-"$(npm bin)"/lint-staged
+yarn lint-staged
```

Comment:
You could have used `npx` to access node_module binaries, and so avoiding the usage of yarn 🤔 

```
npx lint-staged
```

Reply:
`npx` has bad side effects when `node_modules` aren't fully installed.  `npm exec` is the equivalent but using `yarn` here is intentional.  We use `yarn` to install dependencies 🤷